index.css

```diff
@@ -421,4 +439,54 @@ li {
     line-height: 16px;
     color: #344563;
   }
+
+  #contact-me {
+    background-color: #6070ff;
+    padding: 50px 24px 30px 24px;
+    background-image: url("./images/main-bg.png");
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a design template precisely. Let's work on the background image to match the design. It's looking a bit off when compared to the Figma template. 🤓️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220433277-e02a20f4-2997-47e4-91c0-dcea943652cf.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220433284-381f3b86-f53a-44e2-9ff6-8c41622311aa.png" width="300">|

Reply:
fixed the contact-me section background image issue.